sentinelhub/api/batch.py

```diff
@@ -456,16 +462,7 @@ def _get_collections_url(self, collection_id: Optional[str] = None) -> str:
             return f"{url}/{collection_id}"
         return url
 
-    @staticmethod
-    def _parse_request_id(data: BatchRequestType) -> str:
-        """Parses batch request id from multiple possible inputs"""
-        if isinstance(data, BatchRequest):
-            return data.request_id
-        if isinstance(data, dict):
-            return data["id"]
-        if isinstance(data, str):
-            return data
-        raise ValueError(f"Expected a BatchRequest, dictionary or a string, got {data}.")
+    _get_job_endpoint_url = _get_process_url
```

Comment:
I was thinking of the first option, but then didn't decide for it since in batch process there is a distinction between grids, process and collections. However on second look it seems like the renaming wouldn't hurt readability all that much

Reply:
went with second option